packages/components/src/templates/drawer.stories.ts

```diff
@@ -52,31 +53,131 @@ export const Default = {
     <sd-drawer open placement="end">
-      <nav>
-        <sd-navigation-item vertical current><b>Home page</b></sd-navigation-item>
-        <sd-navigation-item vertical chevron divider>About Us</sd-navigation-item>
-        <sd-navigation-item vertical chevron divider>Markets</sd-navigation-item>
-        <sd-navigation-item vertical chevron divider>Press service</sd-navigation-item>
-        <sd-navigation-item vertical chevron divider>Sustainability</sd-navigation-item>
-        <sd-navigation-item vertical chevron divider>Career</sd-navigation-item>
-      </nav>
-
-      <nav slot="footer" class="bg-neutral-100">
-        <sd-navigation-item vertical class="flex align-center">
-          <sd-icon name="system/user" class="h-6 w-6 mr-2"></sd-icon>
-          My depot
-        </sd-navigation-item>
-        <sd-navigation-item vertical divider class="flex align-center">
-          <sd-icon name="system/lock-locked" class="h-6 w-6 mr-2"></sd-icon>
-          My application
-        </sd-navigation-item>
-        <sd-navigation-item vertical divider class="flex align-center">
-          <sd-icon name="system/website" class="h-6 w-6 mr-2"></sd-icon>
-          Our further appearances
-        </sd-navigation-item>
-      </nav>
+      <sd-button slot="header" variant="tertiary" class="hidden" id="returnButton" onclick="handleSecondLevel()">
+        <sd-icon name="system/arrow-left" label="return"></sd-icon>
+      </sd-button>
+      <div class="level-one h-full flex flex-col justify-between">
+        <nav>
+          <sd-navigation-item vertical current><b>Home page</b></sd-navigation-item>
+          <sd-navigation-item vertical chevron divider onclick="handleSecondLevel()">About Us</sd-navigation-item>
+          <sd-navigation-item vertical chevron divider>Markets</sd-navigation-item>
+          <sd-navigation-item vertical chevron divider>Press service</sd-navigation-item>
+          <sd-navigation-item vertical chevron divider>Sustainability</sd-navigation-item>
+          <sd-navigation-item vertical chevron divider>Career</sd-navigation-item>
+        </nav>
+        <nav slot="footer" class="bg-neutral-100">
+          <sd-navigation-item vertical class="flex align-center">
+            <sd-icon name="system/user" class="h-6 w-6 mr-2"></sd-icon>
+            My depot
+          </sd-navigation-item>
+          <sd-navigation-item vertical divider class="flex align-center">
+            <sd-icon name="system/lock-locked" class="h-6 w-6 mr-2"></sd-icon>
+            My application
+          </sd-navigation-item>
+          <sd-navigation-item vertical divider class="flex align-center">
+            <sd-icon name="system/website" class="h-6 w-6 mr-2"></sd-icon>
+            Our further appearances
+          </sd-navigation-item>
+        </nav>
+      </div>
+      <div class="level-two hidden">
+        <nav id="level-two-nav">
+          <sd-navigation-item vertical current id="nav-title"><b>About Us</b></sd-navigation-item>
```

Comment:
I'm not sure, if the UX is correct on this one. @MartaPintoTeixeira @auroraVasconcelos 

![CleanShot 2024-10-18 at 12 26 03@2x](https://github.com/user-attachments/assets/f4755e08-727d-428d-835c-dc631938d010)

Instead of a navigation item I expected this to be some kind of headline. AFAIK this is the actual UX pattern.

For comparison see e. g. union-investment.de

![CleanShot 2024-10-18 at 12 27 44@2x](https://github.com/user-attachments/assets/80c93720-27f5-4610-a1e3-0fed69330a0c)

Reply:
Hey @mariohamann I agree with your expectations. Currently it is a nav-item due to the need of the "curent" styles (figma template)[https://www.figma.com/design/YDktJcseQIIQbsuCpoKS4V/Component-Docs?node-id=2228-6162&t=zBzLuKWhp5m7DYeJ-4].
@MartaPintoTeixeira, could we update this?

Reply:
I see your point @mariohamann and understood. Lets change it.

Reply:
Updated in code now